compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -271,6 +276,8 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
       // transpose.
       .addPredicatedPass(clNormalizeInputIndexingMap,
                          createInterchangeTransposeGenericOpsPass)
+      .addPredicatedPass(clEnableConvToWinograd,
```

Comment:
Please move this into `buildOptionalPreprocessingPassPipeline`.

Reply:
sure will move